font.go

```diff
@@ -161,6 +164,14 @@ func (f *fontImpl) SimpleWidth(str string) float32 {
 	return skia.FontMeasureText(f.font, str)
 }
 
+func (f *fontImpl) TextBlobPosH(glyphs []uint16, positions []float32, y float32) TextBlob {
+	builder := skia.TextBlobBuilderNew()
+	skia.TextBlobBuilderAllocRunPosH(builder, f.font, glyphs, positions, y)
+	blob := skia.TextBlobBuilderMake(builder)
+	skia.TextBlobBuilderDelete(builder)
+	return *newTextBlob(blob)
```

Comment:
You shouldn't dereference here and instead should pass around the pointer everywhere since this contains a reference to a C object. You run the risk of prematurely undef'ing the object with the way it is now.